deployment/sriov-network-operator/templates/configmap.yaml

```diff
@@ -20,4 +20,5 @@ data:
   Nvidia_mlx5_MT42822_BlueField-2_integrated_ConnectX-6_Dx: "15b3 a2d6 101e"
   Broadcom_bnxt_BCM57414_2x25G: "14e4 16d7 16dc"
   Broadcom_bnxt_BCM75508_2x100G: "14e4 1750 1806"
-  Qlogic_qede_QL45000_50G: "1077 1654 1664"
\ No newline at end of file
+  Qlogic_qede_QL45000_50G: "1077 1654 1664"
+  
```

Comment:
and here

Reply:
Any thoughts I have tried multiple combination error still remains ?
https://user-images.githubusercontent.com/55138250/158124068-a51f6746-e9fd-4631-a166-64b5dcea3136.png